motion/ruby_motion_query/stylers/ui_button_styler.rb

```diff
@@ -52,6 +52,26 @@ def background_image_highlighted=(value)
         @view.setBackgroundImage value, forState: UIControlStateHighlighted
       end
 
+      def background_image_selected=(value)
+        @view.setBackgroundImage value, forState: UIControlStateSelected
```

Comment:
you mentioned only 2 can be tested, is this the one that was left out?  can't we test it like the other image tests?
```ruby
view.imageForState(UIControlStateSelected).should == @vc.rmq.image.resource('logo')
```